### ISSUE TITLE
Show intermediate test results in the panel

### ISSUE THREAD

**Issue by rwols** [SublimeText/UnitTesting#186]
The panel remains empty while the tests are running. It would be nice to print the intermediate test results while the test runner is running.

**Comment by randy3k** [comment]
The panel should be updated incrementally. Is it!?

**Comment by rwols** [comment]
I don't see any incremental updates. ST 4084 on Linux.

**Comment by randy3k** [comment]
Not sure if you mean this.

![Sep-03-2020 15-52-43](https://user-images.githubusercontent.com/1690993/92181179-8b6e6c80-edfd-11ea-8ae2-2d00213171d7.gif)

ST 4084 on macOS.

**Comment by randy3k** [comment]
By the way, are you using deferred testcases?

**Comment by rwols** [comment]
Yes. A mixture of regular `unittest.TestCase` and `unittesting.DeferredTestCase` even.

**Comment by gerardroche** [comment]
I've been updating NeoVintageous to Python 3.8 in ST4 and have similar issues. 

I've narrowed the issue down to the UnitTesting reloader. The reloader causes random failures. 

In NeoVintageous ([st4 branch](https://github.com/NeoVintageous/NeoVintageous/tree/st4)) the tests randomly fail and the test panel can remain empty like described above. 

When I comment out the reloading code it works, but obviously I need to restart ST when I make changes to the plugin:

https://github.com/gerardroche/UnitTesting/commit/dcfec0b4878064fe6e29b8fca706c497b0eb9fc2

With the above change the NeoVintageous tests run fine, .

For my [PHPUnitKit](https://github.com/gerardroche/sublime-phpunit) plugin I replaced the reloader code with (AutomaticPackageReloader plugin):

```
sublime.active_window().run_command('package_reloader_reload')
```

That seemed to work for PHPUnitKit but not for NeoVintageous.
